web_src/js/components/ActivityHeatmap.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="user-heatmap">
     <div class="total-contributions">
-      {{ sum }} contributions in the last 12 months
+      {{ sum }} {{ locale.contributions_in_the_last_12_months }}
```

Comment:
That seems incorrect.
```suggestion
      {{ sum }} {{.locale.Tr locale.contributions_in_the_last_12_months }}
```
(Hopefully, this works. I know that Go templates are weird…)

Reply:
It's `vue`, will go templating be parsed here?

Reply:
Yep, ignore this then.